wdtk-client/src/test/java/org/wikidata/wdtk/client/JsonSerializationActionTest.java

```diff
@@ -60,6 +60,7 @@ public void testDefaults() {
 		assertTrue(action instanceof JsonSerializationAction);
 		assertFalse(action.needsSites());
 		assertTrue(action.isReady());
+		assertEquals(action.getActionName(), "JsonSerializationAction");
 	}
 
```

Comment:
Currently we are missing tests that test the actual report output. You are testing the report string, but not its writing to file. This could be added below by setting a report file for one of the tests, and making sure that it contains a line starting with the action name. It would also be useful to set a non-standard action name to test this in action as well. If you do this in only one of the methods that test JSON output, both report file and logger report will be tested.